types/next/test/next-dynamic-tests.tsx

```diff
@@ -53,7 +69,7 @@ const helloBundleJSX = <HelloBundle foo="bar" />;
 
 // 5. With plain Loadable options
 const LoadableComponent = dynamic({
-    loader: () => asyncComponent,
+    loader: () => import('./imports/with-default'),
```

Comment:
I believe Next.js examples have changed - before were using the component promises directly instead of the loader functions. Do you know if that's required now? Or if it accepts both?